use-cases/financial_payment_classification/financial payment classification.ipynb

```diff
@@ -0,0 +1,1849 @@
+{
```

Comment:
<p>Can you estimate how long "a while" is so customers have an expectation of how long they'll wait? 1 minute, 5 minutes, ...? Based on the print statements below, it looks like it takes &lt; 1 minute.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3017/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
solved. rephrased the sentence